api/models.py

```diff
@@ -5,10 +5,10 @@
 class UserProfile(models.Model):
 
 	ACTIVITY_LEVEL = (
-			("SC", "Sleepy_Cat"),
-			("LC", "Lazy_Cat"),
-			("AC", "Active_Cat"),
-			("MC", "Mentor_Cat"),
+			("04", "Sleepy_Cat"),
```

Comment:
These changes make it much easier to sort members by group on the page. It is not such a big change, I think. If noone strongly objects, I suggest we change these values in the model. If can not change these, then will need to figure out how to sort better on the member page.